playbooks/roles/load-balancer-v2/templates/backend.map.ctmpl

```diff
@@ -1,4 +1,12 @@
-#jinja2:variable_start_string:'{{!', variable_end_string:'!}}'
+#jinja2:variable_start_string:'{{!', variable_end_string:'!}}', trim_blocks:False
+{{- /* Custom haproxy maps (written first, to take precedence over automated Ocim entries.). */ -}}
```

Comment:
@giovannicimolin, how about this suggestion?

```suggestion
{{- /* Custom haproxy maps (written first, to take precedence over automated Ocim entries in case there are same domains used there.). */ -}}
```